newdle/client/src/locales/es/messages.po

```diff
@@ -13,6 +13,10 @@ msgstr ""
 "Language-Team: \n"
 "Plural-Forms: \n"
 
+#: src/components/NewdlesParticipating.js:28
+msgid "<0>You are not part of any newdles yet.</0>"
```

Comment:
just fyi, the `<0>...</0>` around this will disappear once you fix the nesting with h2/trans